### ISSUE TITLE
Gutenframe: Verify editing flows on the desktop app.

### ISSUE THREAD

**Issue by kwight** [Automattic/wp-calypso#32048]
We'll need to verify that editing flows all work as expected when working with Atomic and Jetpack sites within the WordPress.com Desktop app.

Background: p3fqKv-6Hh-p2.

**Comment by codebykat** [comment]
Okay so I spent some time on this but wasn't able to figure out how to compile the desktop app against a Calypso beta.

Steps I followed:
* I created #32673 to enable the feature flags in the desktop environments.
* I installed and built the desktop app locally, following the instructions [here](https://github.com/Automattic/wp-desktop/blob/b729377828ec53f4469ed27a15c1fbc6e9b3a1cf/docs/install.md). Verified I could create a working build.
* Based on the testing instructions in #31580, I checked out the Calypso branch within the desktop repo, and used `make build` again.

Results:
* With Calypso running locally from the wp-calypso repo on calypso.local:3000, with the above branch checked out: Unable to opt in to the block editor on any Jetpack site.
* With Calypso running locally from `make dev-server` in the desktop repo (as per [here](https://github.com/Automattic/wp-desktop/blob/develop/docs/development.md)):

<img width="1044" alt="Screen Shot 2019-04-29 at 10 40 12 PM" src="https://user-images.githubusercontent.com/52152/57038949-163ec600-6c10-11e9-9f93-16ccb7fa4058.png">

Kicking this back to the backlog for now, maybe someone else can get a working build going.

@mmtr did some sleuthing and asked for help [here](https://github.com/Automattic/wp-calypso/pull/32673#issuecomment-488218505).
